### DISCUSSION TITLE
Need to make all dashboard text Dynamic

### DISCUSSION THREAD

**Discussion by sonaliTekdi** [Sunbird-cQube/community#810] [Q&A]
We are going to make the Home component's text dynamic (e.g., 'Welcome to NDEAR VIDYA SAMIKSHA KENDRA. Please select your role.'). As we can see, there is already a ui_config.json JSON file containing dynamic text configurations. Can I add more texts to that file and reflect the changes in the Home component, similar to the layout component?

**Comment by FaizanMohammed326** [comment]
Yes definitely you can, you just have to import that ui_config.json and use variables accordingly  in the home component

**Comment by sonaliTekdi** [comment]
Thank You.
